op-challenger/game/fault/responder/responder_test.go

```diff
@@ -289,3 +290,7 @@ func (m *mockContract) UpdateOracleTx(_ context.Context, claimIdx uint64, data *
 	m.updateOracleArgs = data
 	return txmgr.TxCandidate{TxData: ([]byte)("updateOracle")}, nil
 }
+
+func (m *mockContract) GetRequiredBond(_ context.Context, position types.Position) (*big.Int, error) {
+	return big.NewInt(5), nil
+}
```

Comment:
We should add a test here that an error returned by fetching the required bond is bubbled up by `PerformAction`. Can do in a follow-up pr if you agree

Reply:
Yeah can follow up on this!